config/v1/types_feature.go

```diff
@@ -95,7 +109,9 @@ type FeatureGateEnabledDisabled struct {
 //
 // If you put an item in either of these lists, put your area and name on it so we can find owners.
 var FeatureSets = map[FeatureSet]*FeatureGateEnabledDisabled{
-	Default: {
+	Default:         defaultFeatureGates,
+	CustomNoUpgrade: defaultFeatureGates,
```

Comment:
I find this suspicious.  I think custom should be custom.  Maybe they want to adjust something we set by default. 

Please remove the change from this pull

Reply:
Fixed the CustomNoUpgrade up. Thanks